gke-disk-image-builder/cli/main.go

```diff
@@ -37,10 +38,15 @@ func (s *stringSlice) Set(value string) error {
 	return nil
 }
 
+var (
+	gcpResourceNameRegex = regexp.MustCompile("^[a-z]([-a-z0-9]*[a-z0-9])?$")
```

Comment:
Where did you find the rules for GCP resource names?

Reply:
https://cloud.google.com/compute/docs/naming-resources

The regex is here: https://screenshot.googleplex.com/5H4GFNpxnDabQJX